codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/AuthGenerator.java

```diff
@@ -91,6 +115,13 @@ public void renderRbs() {
         LOGGER.fine("Wrote auth rbs module to " + rbsFile());
     }
 
+    private Object removeRbExtension(String s) {
+        if (s != null && s.endsWith(".rb")) {
+            return s.split(".rb")[0];
+        }
+        return s;
+    }
+
     private void renderAuthParamsClass(RubyCodeWriter writer) {
         // TODO: this should have more params when hooked up with endpoint/auth rules?
```

Comment:
Should we be adding the additionalAuthParams from the authSchemes here?

Reply:
Yeah. It's on my plan. I also need signer properties somehow.